tests/foreman/cli/test_repository.py

```diff
@@ -691,3 +699,162 @@ def test_positive_upload_content(self):
             "Successfully uploaded file '{0}'".format(RPM_TO_UPLOAD),
             result[0]['message'],
         )
+
+
+class RepositoryExportTestCase(CLITestCase):
+    """Tests for exporting a repository via CLI"""
+
+    def setUp(self):
```

Comment:
@abalakh about the usage of plain assert, there are some test runners that don't present good messages when it's fail.

You can run those on setUp and then assign the result on a class atrribute, like:

```python
class MyTestCase(TestCase):
    is_setup = False
    def setUp(self):
        if not MyTestCase.is_setup:
            MyTestCase.something = do_something()
            self.assertTrue(MyTestCase.something)
            MyTestCase.is_setup = True
```